geopandas/__init__.py

```diff
@@ -23,7 +23,6 @@
 import pandas as pd  # noqa
 import numpy as np  # noqa
 
-from ._version import get_versions
+from . import _version
 
-__version__ = get_versions()["version"]
-del get_versions
+__version__ = _version.get_versions()["version"]
```

Comment:
Is this change manual or is it done by the bot? I.e. is this something we need to manually fix in the PR with every update?

Reply:
The bot added the section:
```Python

from . import _version

__version__ = _version.get_versions()["version"]
```
But a similar section was currently already in `__init__.py`:
```Python
from ._version import get_versions

__version__ = get_versions()["version"]
del get_versions
```
So I removed the latter section. I suspect the bot will check next time for the former, and since that's already there it shouldn't add it again.

If I'm right, this is a one time change only for this PR, and upcoming PRs don't need that manual change.